core/state/node/migrations/0000000001-init.json

```diff
@@ -80,7 +80,7 @@
                         }
                     }
                 },
-                "additionalProperties": false,
+                "additionalProperties": true,
```

Comment:
What was this blocking before?

Reply:
trying to change something from a pointer to not a pointer in the underlying golang struct